ecosystem/typescript/sdk/src/token_client_abis.ts

```diff
@@ -3,6 +3,12 @@
 
 /* eslint-disable max-len */
 
+// The Token ABIs are generated through Move compilation. We first need to compile the token smart contract. And then,
+// we hex-code the ABI file contents.
+
+// See https://aptos.dev/sdks/transactions-with-ts-sdk#build-raw-transactions-with-the-abi-transaction-builder for more
```

Comment:
we're probably going to delete this from the doc site, it belongs in the sdk itself.